### PR TITLE
Streamline CI reporting to PRs

### LINKED ISSUE #2007: Clean up CI user reporting
### What new functionality do you need?

The user output reporting to GitHub messages for a CI test in a PR needs improvement:

- A more judisous use of the word  *** Sucsess *** (used too often and hinders failure messaging)
- Reduce the frequency of reports and needs be more holistic
- Maybe have a way to report status for jobs that have been running exceptionally long

### What are the requirements for the new functionality?

The output looks better

### Acceptance Criteria

A followup critique from global-workflow branch managers   

### Suggest a solution (optional)

_No response_

### PR DESCRIPTION
**Pull request by TerrenceMcGuinness-NOAA**
# Description

**Cleaned up the logs for CI PRs sent to the GitHub Messages:**

- Eliminates redundant accumulative reporting
- Reports on all created or skipped experiment instantiations in a single message
- Gives one line report once  for each experiment completion
- Single final report completion of CI

See some examples in forked [PR 195](https://github.com/TerrenceMcGuinness-NOAA/global-workflow/pull/195)

Resolves #2007

# Type of change
- Maintenance (**clean-up**)
# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO
# How has this been tested?
- Cycled test on Orion in dev CRON CI jobs
# Checklist
- [ ] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: ci/scripts/clone-build_ci.sh [hunk 1]
```diff
@@ -78,19 +70,17 @@ echo "${commit}" > "../commit"
 # run checkout script
 cd sorc || exit 1
 set +e
-# TODO enable -u later when GDASApp tests are added
 ./checkout.sh -c -g -u >> log.checkout 2>&1
 checkout_status=$?
 if [[ ${checkout_status} != 0 ]]; then
   {
-    echo "Checkout:                      *FAILED*"
+    echo "Checkout: *** FAILED ****"
```
Comment (WalterKolczynski-NOAA) [inline ci/scripts/clone-build_ci.sh]:
```suggestion
    echo "Checkout: *** FAILED ***"
```

File: ci/scripts/clone-build_ci.sh [hunk 2]
```diff
@@ -104,19 +94,27 @@ build_status=$?
 
 if [[ ${build_status} != 0 ]]; then
   {
-    echo "Build:                         *FAILED*"
+    echo "Build: ** FAILED ***"
```
Comment (WalterKolczynski-NOAA) [inline ci/scripts/clone-build_ci.sh]:
```suggestion
    echo "Build: *** FAILED ***"
```

File: ci/scripts/clone-build_ci.sh [hunk 3]
```diff
@@ -104,19 +94,27 @@ build_status=$?
...
   } >> "${outfile}"
   exit "${build_status}"
 else
   {
-    echo "Build:                         *SUCCESS*"
     echo "Build: Completed at $(date)" || true
   } >> "${outfile}"
 fi
 
+export MACHINE_ID="orion"
```
Comment (WalterKolczynski-NOAA) [inline ci/scripts/clone-build_ci.sh]: Suspect this shouldn't be hard-coded here. Isn't it already set by a detect_machine call?

**Comment by WalterKolczynski-NOAA** [comment]
@TerrenceMcGuinness-NOAA Would you please update the PR to describe what the changes are and what the intended purpose is? Bonus points if you include a sample of the new output.

A: Done ~T.McG :)

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]

File: ci/scripts/clone-build_ci.sh (hunk 3 above)
Comment (TerrenceMcGuinness-NOAA) [reply ci/scripts/clone-build_ci.sh]: Oops!!  Yes of course,  that was there for developing on workstation.

**Review by TerrenceMcGuinness-NOAA — COMMENTED** [review]
Resolved all comments

**Review by TerrenceMcGuinness-NOAA — COMMENTED** [review]
committing comment

File: ci/scripts/check_ci.sh [hunk 4]
```diff
@@ -145,16 +146,16 @@ for pr in ${pr_list}; do
...
-      sed -i "s/\`\`\`//2g" "${GFS_CI_ROOT}/PR/${pr}/output_${id}"
-      "${GH}" pr comment "${pr}" --repo "${REPO_URL}" --body-file "${GFS_CI_ROOT}/PR/${pr}/output_${id}"
       #Remove Experment cases that completed successfully
       rm -Rf "${pslot_dir}"
       rm -Rf "${pr_dir}/RUNTESTS/COMROT/${pslot}"
+      rm -f "${output_ci_single}"
+      echo "\`\`\`" > "${output_ci_single}"
+      DATE=$(date)
+      echo "Experiment ${pslot} Completed Successfully at ${DATE}" >> "${output_ci_single}"
+      echo "Experiment ${pslot} Completed Successfully at ${DATE}" >> "${output_ci}"
```
Comment (aerorahul) [inline ci/scripts/check_ci.sh]:
```suggestion
      echo "Experiment ${pslot} **SUCCESS** ${DATE}" >> "${output_ci_single}"
      echo "Experiment ${pslot} **SUCCESS** at ${DATE}" >> "${output_ci}"
```

**Review by aerorahul — COMMENTED** [review]
Looks good.  Just one comment to bold and highlight SUCCESS

**Review by TerrenceMcGuinness-NOAA — COMMENTED** [review]
couple more updates on appearance done

**Review by aerorahul — APPROVED** [review]
looks good.
thanks for shrinking the bot output and making it succinct.

**Review by TerrenceMcGuinness-NOAA — COMMENTED** [review]
Yeah this was much needed updated and Walter had some good suggestions.
```
Testing **bold** in code message
```
Oops, this doesn't bold in `code` messaging.
I prefer 'code' style because it looks like it's coming from a bot.
Maybe we can used regular **bolded** text on the one liners from the completion of experiment messages?
I'll do that and we can change it back latter one we see what it looks like.

**Review by TerrenceMcGuinness-NOAA — COMMENTED** [review]
bold output for success on experiment completion

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Good first step to making the messages more useful.

**Pull request merged**